crates/bevy_asset/src/lib.rs

```diff
@@ -1986,4 +1986,95 @@ mod tests {
 
         app.world_mut().run_schedule(Update);
     }
+
+    #[test]
+    // CAUTION: Due to https://github.com/bevyengine/bevy/issues/18267, the
+    // test is expected to fail. The `#[ignore] should be removed when the
+    // issue is fixed.
+    #[ignore]
```

Comment:
You should move the comment into the ignore reason here and make it a bit terser :) https://doc.rust-lang.org/reference/attributes/testing.html#the-ignore-attribute